ddtrace/contrib/internal/langchain/utils.py

```diff
@@ -0,0 +1,125 @@
+import inspect
+import sys
+
+
+class BaseTracedLangChainStreamResponse:
```

Comment:
We don't need to base this off `wrapt.ObjectProxy`?

Reply:
i had tried that initially, as i took inspiration from the antropic integration. but, when doing that, i end up getting
```
AttributeError: 'generator' object has no attribute '_dd_integration'
```
so i think this is why i went with this approach